sunpy-core.md

```diff
@@ -6,4 +6,8 @@ Planned release by June 2019 with associated paper. The details of this release
 
 ## 2.0
 
+* Create a generic command to read in files such as `sunpy.open()` or `sunpy.read()`
+* Move to use NDCube as base data object in Maps. NDCube object will become the default object returned if a file is not recognized as a defined base data object.
```

Comment:
First sentence is great, second one not so much. There are many questions around how we want to handle >2D data once map is based on NDCube. I think we should aim for just the first one in 2.0 and then think about the extensions once it's been in the wild a little bit.

Reply:
that makes sense. I'll move it and make it more vague.